api/v1alpha1/machine.go

```diff
@@ -92,18 +92,18 @@ type MachineStatus struct {
 	Conditions []MachineCondition `json:"conditions,omitempty"`
 }
 
+// MachineCondition defines an observed condition of a Machine.
 type MachineCondition struct {
 	// Type of the Machine condition.
 	Type MachineConditionType `json:"type"`
 
-	// Status is the status of the Machine condition.
-	// Can be True or False.
+	// Status of the condition.
 	Status ConditionStatus `json:"status"`
```

Comment:
sorry, I'm only seeing this snippet of code, and I'm not familiar with what is going on here, but what does a `MachineCondition.Status` of true or false mean?

Reply:
It depends on the condition - there's 1 condition at present (Contactable) and when set to true it means we can contact the BMC.